examples/intkey/package-lock.json

```diff
@@ -0,0 +1,6810 @@
+{
```

Comment:
I didn't think this file should be checked in.

Reply:
> This file is intended to be committed into source repositories
https://docs.npmjs.com/files/package-lock.json

This goes against the guidance from Node.js, and seems to me to mostly defeat the purpose of a lock file. I disagree entirely. If you are saying it is Sawtooth policy to go against this guidance, then I will remove it in order to comply with that policy.

Reply:
Not checking this in allows us to catch issues that will inevitably crop up during library updates during the CI process.  We should ensure that a nightly build is running for this SDK.

Reply:
There is a Jenkinsfile. @rbuysse would know whether or not a nightly build has been set up.